app/utils/translation.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+def load_translation(language):
+    primary_language = language.split(",")[0].split(";")[0].split("-")[0]
+    print(f"Primary: {primary_language}")
```

Comment:
@ZenVega print statements should be changed for logging messages, if we want to keep them